x-pack/plugins/case/server/routes/api/cases/get_case.ts

```diff
@@ -5,43 +5,42 @@
  */
 
 import { schema } from '@kbn/config-schema';
-import { RouteDeps } from '.';
-import { flattenCaseSavedObject, wrapError } from './utils';
+
+import { CaseResponseRt } from '../../../../common/api';
+import { RouteDeps } from '../types';
+import { flattenCaseSavedObject, wrapError } from '../utils';
 
 export function initGetCaseApi({ caseService, router }: RouteDeps) {
   router.get(
     {
-      path: '/api/cases/{id}',
+      path: '/api/cases',
       validate: {
-        params: schema.object({
-          id: schema.string(),
-        }),
         query: schema.object({
+          id: schema.string(),
```

Comment:
so weird to me but ok

Reply:
matching what has been done in rules API so the user does not get a different feeling when using different API from our team

Reply:
related to this https://github.com/elastic/kibana/pull/59265#discussion_r388528724